### ISSUE TITLE
[BUG] Softlock on "The Ruins"

### ISSUE THREAD

**Issue by Talon1024** [ZDoom/Raze#855]
### Raze version

master b991af2dbc

### Which game are you running with Raze?

Redneck Rampage

### What Operating System are you using?

Linux x86_64

### Please describe your specific OS version

Kubuntu 20.04 LTS

### Relevant hardware info

Intel i5-4690K CPU, NVidia GeForce GTX 970 GPU

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

If the player tries to open the last door on "The Ruins" (RR E2L4) manually without flipping the switch beside it first (which isn't possible without all the keys), the player gets softlocked.

[Here's a saved game in front of the aforementioned door, before trying to open it manually](https://github.com/ZDoom/Raze/files/10416389/ruins.door.zip)

IIRC this is a known bug in the OG, but surely a maphack or something can be added to fix this bug?

### Steps to reproduce the behaviour.

Explain how to reproduce
1. Load the saved game
2. Try to open the door in front of you manually before flipping the switch beside it
3. Once you flip the switch beside it, the game says "Locked!".

**Comment by coelckers** [comment]
Yes, it's a known map bug - but I have no idea how to fix it. Duke/RR's activator system is rather crappy and does not give much control.
